src/pages/ThorChainLP/components/RemoveLiquidity/RemoveLiquidityInput.tsx

```diff
@@ -565,6 +569,7 @@ export const RemoveLiquidityInput: React.FC<RemoveLiquidityInputProps> = ({
       poolAssetGasFeeFiat: poolAssetGasFeeFiat.toFixed(2),
       totalGasFeeFiat,
       feeBps: '0',
+      withdrawalBps: (percentageSelection * 100).toString(),
```

Comment:
Do we want to use `bnOrZero` here? This *should* be safe but floats (which it shouldn't be here as `percentageSelection` is an int but...) in vanilla JS always scare me

Reply:
https://github.com/shapeshift/web/pull/6285/commits/32f59bac73d2f81b3cd1f84ddbbf2c6a3244aa8a